README.md

```diff
@@ -574,19 +587,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ---
 
-**[⬆ Back to Index](#content)**
-
-## Free Hosting
-
-- :wrench: [Netlify](https://www.netlify.com/)
-- :wrench: [Heroku](https://www.heroku.com/)
-- :wrench: [Vercel](https://vercel.com/)
-- :wrench: [Github Pages](http://pages.github.com)
-- :wrench: [Gitlab Pages](http://docs.gitlab.com/ee/user/project/pages)
-- :wrench: [Deta](https://www.deta.sh/)
-- :wrench: [InfinityFree](https://infinityfree.net/)
 
```

Comment:
Remove this line

Reply:
There are many empty lines after the `---` now that the section is moved. There should only be one empty line after it

Reply:
I have resolved it please check and update